### PR TITLE
Request: Add ehashman as SIG Node approver

### PR DESCRIPTION
**Pull request by ehashman**
#### What type of PR is this?

/kind cleanup
/sig node

#### What this PR does / why we need it:

Adds ehashman as a SIG Node approver, ~and cleans up a bunch of node OWNERS files to user the approver alias rather than adding myself to them manually.~ (no longer necessary)

We need more SIG Node approvers!

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

- [x] Reviewer of the codebase for at least 3 months
    - Since January 2021: https://github.com/kubernetes/kubernetes/pull/98036
- [x] Primary reviewer for at least 10 substantial PRs to the codebase
- [x] Reviewed or merged at least 30 PRs to the codebase
   - As of writing I have commented on over 500 SIG Node PRs: https://github.com/kubernetes/kubernetes/pulls?q=is%3Apr+commenter%3Aehashman+label%3Asig%2Fnode
- [x] Nominated by a subproject owner: @derekwaynecarr 
  - With no objections from other subproject owners
  - [ ] @Random-Liu 
  - [ ] @dchen1107 
  - [x] @derekwaynecarr 
  - [ ] @yujuhong 
  - [x] @sjenning 
  - [x] @mrunalp 
  - [x] @klueska 
  - Done through PR to update the top-level OWNERS file: this PR

/cc @Random-Liu @yujuhong @sjenning @mrunalp @klueska 
/assign @dchen1107 @derekwaynecarr 

~/assign @tallclair 
I removed you from some OWNERS files as an approver per discussion on Slack.~

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by sjenning** [comment]
I support @ehashman for sig-node approver.  In her relatively short time in sig-node, she has contributed significantly to both sig-node planning and enhancement/code review.

**Comment by mrunalp** [comment]
:+1:

**Comment by derekwaynecarr** [comment]
+1

look forward to your continued collaboration in SIG.

**Comment by dims** [comment]
+1 from me (non-binding!)

**Comment by haircommander** [comment]
big +1, very well deserved (non-binding)

**Comment by xiaoxubeii** [comment]
👍

**Comment by klueska** [comment]
+1 (binding)

**Comment by odinuge** [comment]
+1 (non-binding)

**Comment by tallclair** [comment]
lgtm for owners changes. Thanks!

**Comment by rphillips** [comment]
+1 (non-binding)

/lgtm

File: pkg/kubelet/cm/OWNERS [hunk 1]
```diff
@@ -1,12 +1,7 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- Random-Liu
-- dchen1107
-- derekwaynecarr
-- vishh
-- yujuhong
+- sig-node-approvers
 - ConnorDoyle
```
Comment (kad) [inline pkg/kubelet/cm/OWNERS:5]: this line seems to be forgotten or left intentionally? or should be moved to emeritus_approvers ?

Comment (ehashman) [reply pkg/kubelet/cm/OWNERS:5]: Left intentionally, @ConnorDoyle is not a top-level node approver.

**Comment by pacoxu** [comment]
+1
Got a lot of guides and help from you in sig-node-related issues/PRs. Really appreciate your work.

**Comment by matthyx** [comment]
+1 @dchen1107 we really need more approvers, please!

**Comment by dims** [comment]
+1 from me (non-binding!)

**Comment by adisky** [comment]
Big +1 for always helping out node stuff!!

**Comment by egernst** [comment]
SGTM

**Comment by troy0820** [comment]
+1 from me (non-binding)

**Comment by wzshiming** [comment]
+1 from me (non-binding)

**Comment by gjkim42** [comment]
+1 from me (non-binding)

**Comment by dims** [comment]
Assigning to @Random-Liu @yujuhong @dchen1107 as they haven't chimed in yet. This has been a while, can you please leave a note here one way or another?

**Comment by dchen1107** [comment]
Sorry for not updating the PR with the latest status here. 

We discussed this at SIG Node meeting several times in year 2021 ([1](https://docs.google.com/document/d/1Ne57gvidMEWXR70OxxnRkYquAoMpt56o75oZtg-OeBg/edit#bookmark=id.wcidhe8lp6zv) ,[2](https://docs.google.com/document/d/1Ne57gvidMEWXR70OxxnRkYquAoMpt56o75oZtg-OeBg/edit#bookmark=id.cl8p5mqtaguc) [3](https://docs.google.com/document/d/1Ne57gvidMEWXR70OxxnRkYquAoMpt56o75oZtg-OeBg/edit#bookmark=id.ns3l0kyb0nnh)), and the community felt that we don't have a clear criteria written in doc for SIG Node reviewers and approvers. 

Several approvers also felt both K8s level criteria and SIG Node previous criteria might not apply to today's SIG Node community. The former one is too abstract given the complexity which SIG Node managed, and the community rejected several requests based on SIG Node's previous criteria in the past; the later one is too strict and staled which requires driving several features / projects from end-to-end given today's node components' maturities. We agreed to update criteria based on today's situation, and most of the important is writing them down so that we can apply the same criteria to all applicants. We had a draft written, which is currently under the final review by @derekwaynecarr. I believe Derek did update all those changes with Elana.  Please correct me if I were wrong there, Derek.

Because missing that newly written criteria being published, this request wasn't merged. But the request didn't completely halt. At sig Node, we also agreed that the top level node approval might be too hard and too powerful given the complexity of the system, we decided to break it down to subprojects. Given the big contributions to SIG Node community, @ehashman and @SergeyKanzhelev are the subprojects' approvers. 

All above I said doesn't imply Elana didn't meet the SIG Node's new bar. IMO I think she already did if I remembered the new changes correctly. But we need to publish that criteria first, and Elana need to update this request against the new ones to move forward.

**Comment by dims** [comment]
@derekwaynecarr @dchen1107 +1 to publishing the criteria. Please let me know how i can help get it out quickly. 

@dchen1107 thanks for the full context which was missing before! definitely appreciate that.

**Comment by 249043822** [comment]
+1 from me (non-binding)

**Comment by justaugustus** [comment]
/remove-lifecycle stale

> @derekwaynecarr @dchen1107 +1 to publishing the criteria. Please let me know how i can help get it out quickly.

@derekwaynecarr @dchen1107 -- Do we have an update on publishing SIG Node approver criteria?

This PR has been opened for almost a year and I don't think that's fair to @ehashman.
cc: @kubernetes/steering-committee

**Comment by endocrimes** [comment]
xref https://github.com/kubernetes/community/issues/6612

I've also asked for a final review of the draft document again this week.

**Comment by derekwaynecarr** [comment]
@endocrimes see https://github.com/kubernetes/community/pull/6725

**Comment by ehashman** [comment]
I've reviewed the new SIG Node Contributor Ladder guidelines. I believe that I qualified for approver under these guidelines prior to their drafting and adoption, at the time of my application, and that I continue to qualify.

As I attempted to fill out each bullet point, detailing my contributions to the SIG, I found it increasingly difficult to feel excited about working in a SIG with a contributor ladder whose bar was repeatedly moved shortly after my approver application was submitted. I do not feel that the SIG leadership blocking my application has clearly communicated to me, in public or private, what I needed to do for my application to move forward, and perhaps that is by design. Per the Node Contributor Ladder Guidelines, "No amount of documentation can formalize how this trust is established." ([kubernetes/community/sig-node/sig-node-contributor-ladder.md](https://github.com/kubernetes/community/blob/b1db3eb980036f7a3b953c785b30dd343b037c0e/sig-node/sig-node-contributor-ladder.md))

After taking a period of extended leave from all Kubernetes contributions, I find myself wanting to prioritize work on areas of the project that are open to welcoming new contributors, encouraging their growth, and accepting new leaders.

From the bottom of my heart, I thank everyone in SIG Node and the wider community that supported my application and hope to work with you in other areas of the project!

**Comment by dchen1107** [comment]
@ehashman what you wrote is really unfair to me. I knew you threw dirty on us via social media, but didn't want to do the same attacking you back for a healthy community. 

Let me take this chance to clarify:

* I am not the one against you from day one. The original reason is the folks felt you never drive a big project from end-to-end; The swap support was just started back then. But both Derek and I felt that is unfair or too strict given today's circumstance. I suggested to revise the criteria to recognize you and several other folks' contribution, not simply based on the big projects. 

But we rejected several folks with the old bar in the past, we should make this transparent and inclusive. We immediately discussed this at 6/29/21, and the meeting note is: https://docs.google.com/document/d/1Ne57gvidMEWXR70OxxnRkYquAoMpt56o75oZtg-OeBg/edit#bookmark=id.wcidhe8lp6zv I believe the folks can find the record to watch. 

* Your request was sent out like a negotiation behind the scenes without any space / time to discuss. The folks raised the concerns to us. Should we simply approve all requests like this if the authors scream loudly? 

* The newly written doc was shared with you since August, 2021, and you were invited as one of the editors along with others. Yes, I didn't immediately create one for you because Derek was on leave, and very difficult to get everyone together. Please note that I also have a full time job!

Honestly from the beginning to the end, I only served as the facilitator on the new doc. If you disagreed with something in that doc, you should say so. 

* The criteria doc wasn't waiting for me to approve since August last year. 

* At https://github.com/kubernetes/kubernetes/pull/103122#issuecomment-1298965681, you claimed you believe you met the new bar. But you never updates PR based on the new criteria until today. 

* From day one you joined SIG node, I supported you from all perspectives. We created subproject for you to drive; I participated your leadership program; I met you via 1:1 since I were super busy. I even helped you address the conflicts with others. Even last month, I asked Derek when you are back. I thought the issue can be worked out and conflict can be addressed. 

I really don't think I deserved this! I have been working on K8s before its inception. Last 8 years, I drove so many projects, mentored and helped many folks all over the place. Even last 3 years, I moved on with another full-time stressful job, but still came back to help grow new leads (you were one of them) later because the community needed. 

May I urge you stop attacking me like this?

**Pull request closed without merging**